_episodes/advanced-containers.md

```diff
@@ -75,14 +75,21 @@ We can create a mount between our computer and the running container by using an
 option to `docker container run`. We'll also use the variable `${PWD}` which will substitute
 in our current working directory. The option will look like this
 
-`-v ${PWD}:/temp`
+`--mount type=bind,source=${PWD},target=/temp`
 
 What this means is -- link my current working directory (on the host computer) with the
 container that is about to be started, and inside this container, name the directory `/temp`.
```

Comment:
I wonder if something slightly more descriptive here may be useful? I'm also wondering whether rather than "linking" the directory, a better way of describing this is "making it visible"? Perhaps:

```suggestion
What this means is: make my current working directory (on the host computer) -- the source --
_visible_ within the container that is about to be started, and inside this container, name the
directory `/temp` -- the target.
```